rotkehlchen/api/rest.py

```diff
@@ -3231,7 +3231,19 @@ def _decode_ethereum_transactions(
             tx_hashes: Optional[List[EVMTxHash]],
     ) -> Dict[str, Any]:
         try:
-            self.rotkehlchen.eth_tx_decoder.decode_transaction_hashes(ignore_cache=ignore_cache, tx_hashes=tx_hashes)  # noqa: E501
+            decoded_events = self.rotkehlchen.eth_tx_decoder.decode_transaction_hashes(
+                ignore_cache=ignore_cache,
+                tx_hashes=tx_hashes,
+            )
+            task_manager = self.rotkehlchen.task_manager
+            if tx_hashes is not None and task_manager is not None:
```

Comment:
Add a docstring here. Say that if any events were decoded and they need price queried, schedule the task to do that